eox_nelp/middleware.py

```diff
@@ -131,6 +130,7 @@ def handler(form_instance, form_desc, required=True):
             """Wrapper of https://github.com/eduNEXT/edunext-platform/blob/ednx-release/
             mango.master.nelp/openedx/core/djangoapps/user_authn/views/registration_form.py#L657
             """
-            form_instance._add_field_with_configurable_select_options(field, label, form_desc, required=required)
-
+            form_instance._add_field_with_configurable_select_options(  # pylint: disable=protected-access
+                field, label, form_desc, required=required
```

Comment:
I prefer to split this in multiple lines, but if you don't want to do that, just add the trailing comma

Reply:
[01a2ea4](https://github.com/eduNEXT/eox-nelp/pull/25/commits/01a2ea46a3176b2a6ccf6a7af3dff33ab9fb828c)